ui-components/src/helpers/preferences.tsx

```diff
@@ -225,7 +108,7 @@ export const FormAddress = ({
           </ViewItem>
         </GridCell>
 
-        {type === "residence" && (
```

Comment:
This is the only code change. We were passing in a type prop, but only using one of the values to decide whether or not to show a mailing address checkbox, so now that is a prop specific to this toggle.